### PR TITLE
TRI-1046: Created tests for investigation feature and several minor adjustments

### PR DESCRIPTION
**Pull request by ds-alexander-bulgakov**

### CODE REVIEW

**Comment by ds-jkreutzfeld** [comment]
Tests look good. Do we want to have this in main already? It should probably be merged to ess-dev at the moment, right?

**Comment by ds-alexander-bulgakov** [comment]
> Tests look good. Do we want to have this in main already? It should probably be merged to ess-dev at the moment, right?

Actually i don´t know. Can´t we merge it to tx-irs since all tavern tests are in this project? What do you think @ds-jhartmann?

File: .github/workflows/tavern.yml [hunk 1]
```diff
@@ -40,6 +40,7 @@ jobs:
       - name: Run tests
         env:
           IRS_HOST: ${{ github.event.inputs.irs-host || 'https://irs.int.demo.catena-x.net' }}
+          IRS_ESS_HOST: ${{ github.event.inputs.irs-ess-host || 'https://irs-ess.dev.demo.catena-x.net' }}
```
Comment (ds-jhartmann) [inline .github/workflows/tavern.yml:43]: Input `irs-ess-host` does not exist. You need to add this to the inputs for `workflow_dispatch` the same way as `irs-host`

**Comment by ds-jhartmann** [comment]
> > Tests look good. Do we want to have this in main already? It should probably be merged to ess-dev at the moment, right?
> 
> Actually i don´t know. Can´t we merge it to tx-irs since all tavern tests are in this project? What do you think @ds-jhartmann?

IMO we don't need scheduled tests for the supplychain feature. At least not until it is part of main. As Jan stated, we should add it to ess-dev. This way we can execute the Tavern tests on demand.

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]

File: .github/workflows/tavern.yml (hunk 1 above)
Comment (ds-alexander-bulgakov) [reply .github/workflows/tavern.yml:48]: changed and commited.

**Comment by ds-alexander-bulgakov** [comment]
I close this PR to avoid branch conflicts. New PR is created for ess-dev branch.

**Pull request closed without merging**